app/views/shared/_output_stream.html.erb

```diff
@@ -1,6 +1,6 @@
 <% if job.active? %>
```

Comment:
So `viewers` tab only works on active jobs. Maybe we hide that tab if job is inactive?